app/index.js

```diff
@@ -70,7 +72,18 @@ app.on('ready', () => {
 
     // config changes
     const cfgUnsubscribe = config.subscribe(() => {
+      const cfg_ = plugins.getDecoratedConfig();
+
       win.webContents.send('config change');
+
+      if (cfg_.shell !== cfg.shell) {
+        notify(
+          'Shell configuration changed!',
+          'Open a new tab or window to start using the new shell'
+        );
+      }
+
+      cfg = cfg_;
```

Comment:
Cool!
What do you think of doing this check here https://github.com/zeit/hyperterm/blob/master/app/config.js#L34 instead?

That way it will only be checked once per load instead of once per window?
(possible that I'm completely wrong)